hosted_control_planes/hcp_high_availability/about-hcp-ha.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hcp-ha"]
+= About high availability for hosted control planes
+include::_attributes/common-attributes.adoc[]
+:context: about-hcp-ha
+
+toc::[]
+
+You can maintain high availability of hosted control planes by implementing the following actions:
```

Comment:
```suggestion
You can maintain high availability (HA) for hosted control planes by implementing the following actions:
```
Not because it's used again, but because I wonder if the "HA" might be more scannable for folks if they're looking for it. I see it like that here: https://docs.openshift.com/serverless/1.33/eventing/tuning/serverless-ha.html